src/model/ShadingControl_Impl.hpp

```diff
@@ -72,6 +72,10 @@ namespace detail {
 
     virtual IddObjectType iddObjectType() const override;
 
+    // If cloning in the same model, will keep on referencing the same SubSurfaces as the original
+    // If cloning in another model, will clear out its extensible Sub Surface Name group
+    virtual ModelObject clone(Model model) const;
```

Comment:
TODO:

* [ ] Decide if this is a valid ShadingControl::clone behavior.

@kbenne @joseph-robertson